postgres-appliance/Dockerfile

```diff
@@ -218,33 +215,41 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 \
     && sed -i "s/ main.*$/ main/g" /etc/apt/sources.list.d/pgdg.list \
     && apt-get update \
-    && apt-get install -y postgresql postgresql-server-dev-all libpq-dev \
+    && apt-get install -y postgresql postgresql-server-dev-all postgresql-all libpq-dev \
     && for version in $DEB_PG_SUPPORTED_VERSIONS; do \
         apt-get install -y postgresql-server-dev-${version}; \
     done \
 \
     && if [ "$DEMO" != "true" ]; then \
         for version in $DEB_PG_SUPPORTED_VERSIONS; do \
-            # due to dependency issues partman has to be installed separately
-            apt-get install -y postgresql-${version}-partman \
+            if [ "$version" != "14" ]; then \
+                # due to dependency issues partman has to be installed separately
+                apt-get install -y postgresql-${version}-partman; \
+            fi \
             # create postgis symlinks to make it possible to perform update
             && ln -s postgis-${POSTGIS_VERSION%.*}.so \
                 /usr/lib/postgresql/${version}/lib/postgis-2.5.so; \
-        done; \
+        done \
+        && PACKAGES="partman repack"; \
     fi \
 \
     # build and install missing packages
-    && for pkg in pgextwlist; do \
-        apt-get source postgresql-$PGVERSION-${pkg} \
+    && for pkg in pgextwlist $PACKAGES; do \
+        apt-get source postgresql-13-${pkg} \
```

Comment:
not $PGVERSION anymore?

Reply:
Only temporary. There is no repack package available for v14.